{"uid":"ca913f2377975d3a","name":"test_address_prefix[Test for KICO]","fullName":"tests.test_network_parameters.TestNetworkPrefix#test_address_prefix","historyId":"c50858c7b873c71d56bcf8d3a386502e","time":{"start":1670012222746,"stop":1670012222945,"duration":199},"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] KICO","statusTrace":"self = <tests.test_network_parameters.TestNetworkPrefix object at 0x7fd512d1c970>\nchain = <tests.utils.chain_model.Chain object at 0x7fd512d79130>\n\n    def test_address_prefix(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:15: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7fd512d79130>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] KICO\n\ntests/utils/chain_model.py:29: TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] KICO","statusTrace":"self = <tests.test_network_parameters.TestNetworkPrefix object at 0x7fd512d1c970>\nchain = <tests.utils.chain_model.Chain object at 0x7fd512d79130>\n\n    def test_address_prefix(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:15: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7fd512d79130>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] KICO\n\ntests/utils/chain_model.py:29: TimeoutError","steps":[],"attachments":[{"uid":"7da0763c478f846","name":"stdout","source":"7da0763c478f846.txt","type":"text/plain","size":27}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_network_parameters"},{"name":"subSuite","value":"TestNetworkPrefix"},{"name":"host","value":"fv-az459-905"},{"name":"thread","value":"1861-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_network_parameters"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"chain","value":"<tests.utils.chain_model.Chain object at 0x7fd512d79130>"}],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"ca913f2377975d3a.json","parameterValues":["<tests.utils.chain_model.Chain object at 0x7fd512d79130>"]}